src/helpers/daily-data-providers/query-garmin-total-daily-values.ts

```diff
@@ -0,0 +1,46 @@
+import { add } from "date-fns";
+import queryAllDeviceData from "./query-all-device-data";
+import { DeviceDataPoint } from "@careevolution/mydatahelps-js";
+
+export default function (types: string[], startDate: Date, endDate: Date, divideBy?: number) {
+	return queryAllDeviceData({
+		namespace: "Garmin",
+		type: "Daily",
+		observedAfter: add(startDate, { days: -1 }).toISOString(),
+		observedBefore: add(endDate, { days: 1 }).toISOString()
+	}).then(function (result) {
+		var data: { [key: string]: number } = {};
+
+		var getProperty = function (d: DeviceDataPoint, type: string) {
+				if (!d.properties) { return; }
+				if (!d.properties[type]) { return; }
+
+				//Garmin average stress level is -2 if you are not wearing the device
+				if (parseInt(d.properties[type]) <= 0) { return; }
+				return Math.round(parseFloat(d.properties[type]));
+		}
+
+		result.forEach((d) => {
+			types.forEach(type => {
+				if (!d.startDate) { return; }
+				if (!d.properties) { return; }
+				if (!d.properties[type]) { return; }
+				//Garmin average stress level is -2 if you are not wearing the device
+				if (parseInt(d.properties[type]) <= 0) { return; }
```

Comment:
Should this be parseFloat?  I guess I dunno what happens if you call parseInt on a float value, presumably since it's JS it'll just parse the int part of it?

Reply:
I brought this part over from the single daily value version of it, so was just keeping consistent here.  But yes, it will just parse the integer portion.  Amsuingly you can do something as strange as `parseInt("4.foo")` and you'll get the value `4` out of it.